modules/hcp-topology-spread-constraint.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-manage/hcp-manage-virt.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-topology-spread-constraint_{context}"]
+= Spreading node pool VMs by using topologySpreadConstraint
+
+By default, KubeVirt virtual machines (VMs) created by a node pool are scheduled to any available node with the capacity to run the VM. When node pool VMs are created, the default `topologySpreadConstraint` constraint is set to schedule VMs on multiple nodes.
+
+In some scenarios, VMs of a node pool might run on the same node which can cause an availability issue. To avoid distribution of VMs on a single node, use the `topologySpreadConstraint` constraint with the descheduler to spread VMs on multiple nodes when needed.
```

Comment:
To avoid distribution of VMs on a single node, use the descheduler to keep honouring the `topologySpreadConstraint` constraint to spread VMs on multiple nodes when needed.